site/app/templates/autograding/Attachments.twig

```diff
@@ -5,10 +5,10 @@
             <span class="fas fa-plus-circle" style='vertical-align:text-bottom;'></span>
             {{ file.name }}</a> &nbsp;
         <a id = 'open_file_{{ file.name|url_encode }}' onclick='popOutSubmittedFile("{{ file.name|url_encode }}", "{{ file.path|url_encode }}")' aria-label="Pop up the file in a new window" class="key_to_click" tabindex="0"><i class="fas fa-window-restore" title="Pop up the file in a new window"></i></a>
-        <a onclick='downloadFile("{{ file.path|url_encode }}", "attachments")' aria-label="Download the file" class="key_to_click" tabindex="0"><i class="fas fa-download" title="Download the file"></i></a>
+        <button onclick='downloadFile("{{ file.path|url_encode }}", "attachments")' aria-label="Download the file" class="key_to_click" tabindex="0"><i class="fas fa-download" title="Download the file"></i></button>
         {% if can_modify %}
-        <a onclick='deleteAttachment(this, "{{ file.name|url_encode }}")' aria-label="Delete the file" class="key_to_click" tabindex="0"><i class="fas fa-trash" title="Delete the file"></i></a>
+        <button onclick='deleteAttachment(this, "{{ file.name|url_encode }}")' aria-label="Delete the file" class="key_to_click" tabindex="0"><i class="fas fa-trash" title="Delete the file"></i></button>
```

Comment:
You should also be able to delete the `key_to_click` class since it was only used to add an event listener to manually trigger the `onclick` event on `a` tags. But the button element will automatically fire it 

see this function here which does the logic for adding the events
https://github.com/Submitty/Submitty/blob/master/site/public/js/server.js#L1414-L1424

Reply:
I will do as suggested.